pom copy.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
and what is this ?

Reply:
> and what is this ?

Sorry, it was accidently added to the pull request. I have removed it.